garnet.json

```diff
@@ -0,0 +1,28 @@
+{
+	"response": [
+		{
+			"maintainer": "JYR_RC",
+			"oem": "Redmi",
+			"device": "Redmi Note 13 Pro 5G",
+			"filename": "crDroidAndroid-15.0-20241110-garnet-v11.0-BETA.zip",
+			"download": "https://sourceforge.net/projects/crdroid/files/garnet/11.x/crDroidAndroid-15.0-20241110-garnet-v11.0-BETA.zip/download",
+			"timestamp": 1731239466,
+			"md5": "0c77964f0a8d131051bba9dbf0901985",
+			"sha256": "efe97cb5ef0b02df502f8f77f4fdab6d8cc1fe62ea57b2bba64ba8754477db1e",
+			"size": 1886000690,
+			"version": "11.0",
+			"buildtype": "BETA",
+			"forum": "https://xdaforums.com/t/rom-android-15-redmi-note-13-pro-5g-crdroid-v11-x-official-monthly-beta.4702220/",
```

Comment:
fix support link

Reply:
I'm going to do it, I didn't realize,sorry